### DISCUSSION TITLE
Help about some things

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#79] [Q&A] [answered]
Hi,
I have some issues.
I tried to disable some plugins in the '[...]/lua/config/lazy.lua' file, but it doesn't work.
In particular, which-key plugin remapped all the key, so in Normal mode I can't use basic command like 'x', 'y', etc.

The file $HOME/.local/share/nvim/lazy/LazyVim/lua/lazyvim/config/keymaps.lua have these:
```
vim.keymap.set("n", "j", "v:count == 0 ? 'gj' : 'j'", { expr = true, silent = true })
vim.keymap.set("n", "k", "v:count == 0 ? 'gk' : 'k'", { expr = true, silent = true })
```
It doesn't work because I see both the string 'gj' and 'gk' in the Lualine statusline.

About Lualine: how can I remove the time?

Every time I start nvim, I got:
"failed to install "

Watching ':MasonLog', I noticed the following:
```
[ERROR ...] ...l/share/nvim/lazy/mason.nvim/lua/mason-registry/init.lua:67: Failed to load package "jsonls": "bad argument #1 to '?' (string expected
```

TIA.
Regards

**Comment by folke** [comment]
Can you please at least provide your actual config? There's literally nothing here I can use.

**Reply by ghost** [reply]
My config is the default, I haven't changed anything.

**Comment by folke** [comment, marked as answer]
which-key does not overwrite those keys you are talking about.

> It doesn't work because I see both the string 'gj' and 'gk' in the Lualine statusline.

If you see that then it does work. What do you mean?

To disable plugins, check the examples. Don't do this in `lua/config/lazy.lua`.
Create a file `lua/plugins/whatever.lua` instead and add it in there.

Check `:Mason` to see if jsonls is effectively installed. Also check with `:checkhealth mason`

lualine: check the lualine docs, but to remove the clock:

```lua
  -- lualine
  {
    "nvim-lualine/lualine.nvim",
    event = "VeryLazy",
    opts = function(_, opts)
      opts.sections.lualine_z = {}
    end,
  },
```

**Reply by ghost** [reply]
About which-key: as I said, in normal mode I can't use those keys: if I select a text by 'v', when I press 'x' doesn't work. Again: if I press 'v', which-key menu is displayed.

**Reply by ghost** [reply]
Is there a way to disable it?

**Reply by folke** [reply]
Did you actually read what I said?

It's all in the docs. Check the examples.

Also, you are clearly not using the default settings since you get an error about jsonls and you mentioned that you could not disable a plugin

**Reply by ghost** [reply]
Uhm, it seems my English is not clear (I know, it's not my native language)

**Reply by MarWeUMR** [reply]
> if I select a text by 'v', when I press 'x' doesn't work. 

I am pretty sure, that this particular issue goes away if you disable the `leap` plugin, since it remaps `x` to `d` in visual mode. 
This tricked me at first aswell. 

> Again: if I press 'v', which-key menu is displayed.

The which-key popup is controlled by `vim.opt.timeoutlen = 300`, if you would bump that up to say 3000ms, it doesn't show up so fast and you would see, that `v` works to select text. With the default setting it works, too - but if you don't select text right away the popup will appear. That shouldn't interfere with selecting anything though...

**Reply by ghost** [reply]
Hi, thank you, I will try. Within ~/.config/nvim/init.vim, I have `set timeoutlen=500`.
